core/src/main/java/co/infinum/goldfinger/CancellableAuthenticationCallback.java

```diff
@@ -3,22 +3,23 @@
 import android.support.v4.hardware.fingerprint.FingerprintManagerCompat;
 import android.support.v4.os.CancellationSignal;
 
+import static co.infinum.goldfinger.LogUtils.log;
+
 class CancellableAuthenticationCallback extends FingerprintManagerCompat.AuthenticationCallback {
 
     private static final long IGNORE_CANCEL_MS = 100;
 
+    final CancellationSignal cancellationSignal;
+
+    private final Goldfinger.Callback callback;
+    private final Clock clock;
     private final Crypto crypto;
-    private final Logger logger;
+    private final long initializationTime;
```

Comment:
I think it would be wise to name this variable based on its unit. For example `initializationTimeMills` if the unit is milliseconds. As soon as I saw it I was wondering what's the unit since it's a time variable and I had to search for it in the code. This is not wrong, I would just like it to be more explicit.